src/evm-block-extractor/src/database/postgres_db_client.rs

```diff
@@ -237,6 +240,94 @@ impl DatabaseClient for PostgresDbClient {
             .map_err(|e| anyhow::anyhow!("Error getting transaction {}: {:?}", hex_tx_hash, e))
             .and_then(|row| from_row_value(&row, 0))
     }
+
+    async fn discard_blocks_starting_with(
```

Comment:
starting_with gives more an idea of a "prefix" more than a range. I would rename to discard_blocks_from or since

Reply:
Agree. Done.